[2024-07-20T00:00:05.497+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-20T00:00:05.530+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.ft_posting_f scheduled__2024-07-19T00:00:00+00:00 [queued]>
[2024-07-20T00:00:05.544+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.ft_posting_f scheduled__2024-07-19T00:00:00+00:00 [queued]>
[2024-07-20T00:00:05.545+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-07-20T00:00:05.565+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): ft_posting_f> on 2024-07-19 00:00:00+00:00
[2024-07-20T00:00:05.604+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=4245) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-20T00:00:05.616+0000] {standard_task_runner.py:64} INFO - Started process 4253 to run task
[2024-07-20T00:00:05.637+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'ft_posting_f', 'scheduled__2024-07-19T00:00:00+00:00', '--job-id', '373', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpplix6ctl']
[2024-07-20T00:00:05.640+0000] {standard_task_runner.py:91} INFO - Job 373: Subtask ft_posting_f
[2024-07-20T00:00:05.822+0000] {task_command.py:426} INFO - Running <TaskInstance: insert_data.ft_posting_f scheduled__2024-07-19T00:00:00+00:00 [running]> on host 04b0ec8dd5cd
[2024-07-20T00:00:05.959+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sfakhrutdinov' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='ft_posting_f' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-19T00:00:00+00:00'
[2024-07-20T00:00:05.964+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-20T00:00:06.054+0000] {base.py:84} INFO - Using connection ID 'postgres-db' for task execution.
[2024-07-20T00:00:07.146+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-20T00:00:07.148+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-20T00:00:07.154+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=ft_posting_f, run_id=scheduled__2024-07-19T00:00:00+00:00, execution_date=20240719T000000, start_date=20240720T000005, end_date=20240720T000007
[2024-07-20T00:00:07.171+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-20T00:00:07.209+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-20T00:00:07.210+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
